packages/backend/src/tokens/__tests__/request.test.ts

```diff
@@ -299,6 +298,7 @@ export default (QUnit: QUnit) => {
         mockRequestWithCookies(
           {
             ...defaultHeaders,
+            'sec-fetch-dest': 'empty',
```

Comment:
Now that we're defaulting to `sec-fetch-dest: document`, we need to explicitly set it to empty for this test case